tools/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.1
 
 require (
 	github.com/cfergeau/gomod2rpmdeps v0.0.0-20210223144124-2042c4850ca8
-	github.com/golangci/golangci-lint v1.59.1
+	github.com/golangci/golangci-lint v1.61.0
```

Comment:
I would keep this commit until the very end, it's odd to have some commits fixing overflows before this one, and then more commits fixing issues after it.

Reply:
Done, push that to end